code/api/src/modules/order/model.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+// Order
+module.exports = function(sequelize, DataTypes) {
+  let Order = sequelize.define('orders', {
+    userId: {
+      type: DataTypes.INTEGER
+    },
+    crateID: {
+      type: DataTypes.INTEGER
+    },
+    deliveryDate: {
+      type: DataTypes.STRING
+    },
+    deliveryStatus: {
+      type: DataTypes.STRING
+    }
+  })
+
+  Order.associate = function(models) {
+    Order.belongsTo(models.User)
+    Order.belongsTo(models.Crate)
+  }
+
```

Comment:
I see that this took a bit more work as Order is new to the database. But same concept as the model for Crate.